src/openrct2/ride/water/SubmarineRide.cpp

```diff
@@ -24,25 +24,16 @@ static uint32_t SubmarineVehicleGetBaseImageId(
     {
         if ((vehicleEntry->sprite_flags & VEHICLE_SPRITE_FLAG_RESTRAINT_ANIMATION) && !(imageDirection & 3))
         {
-            result /= 8;
+            result = OpenRCT2::Entity::Yaw::YawTo4(result);
```

Comment:
I think every unique user of the yawTo functions needs a little explanation. "Restraints are always only in the 4 cardinal directions"

Reply:
This will change in 2 PRs, is it necessary?

Reply:
okay leave for now then.